bin/brew

```diff
@@ -161,7 +168,7 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { "$HOMEBREW_COMMAND" "$@"; exit $?; }
+  { "brew-$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
> Mind naming this `homebrew-$HOMEBREW_COMMAND` to make the namespacing a bit more similar?

I don't mind at all. In fact that is one of the alternatives I also had in mind but for whatever reason I decided to go with the shorter `brew-` prefix. I'll change it to `homebrew-`.